rust/sdk-examples/src/db/common/models/backfill_processor_status.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use crate::{schema::backfill_processor_status, utils::database::DbPoolConnection};
+use diesel::{AsChangeset, ExpressionMethods, Insertable, OptionalExtension, QueryDsl, Queryable};
+use diesel_async::RunQueryDsl;
+
+#[derive(AsChangeset, Debug, Insertable)]
+#[diesel(table_name = backfill_processor_status)]
+/// Only tracking the latest version successfully processed
+pub struct BackfillProcessorStatus {
+    pub processor_name: String,
```

Comment:
Should we name this to be `backfill_alias` ? Same thing with other places where we use `processor_name` instead of `backfill_alias`

Reply:
Good call-- changed.